app/src/main/java/de/tum/in/tumcampusapp/component/ui/chat/ChatMessagesCard.kt

```diff
@@ -74,6 +77,14 @@ class ChatMessagesCard(context: Context,
         putExtras(Bundle())
     }
 
+    override fun getNavigationDestination(): NavigationDestination? {
```

Comment:
I think a non-nullable return type should be a valid override

Reply:
This can’t be non-nullable return type. This replaces `getIntent()`, which is called when the user taps the card. Some cards don’t do anything when tapped, so we need this to be nullable.

Reply:
You need the base method to be nullable, but this overridden method actually does not return null.

Overridden methods *can* return a more precise type than the base method, so a non-null type would be ok here. (aka "compiles for me")
But don't think too much about it, that's not really functionally important :man_shrugging: